### PR TITLE
Replace response.getheader() with response.headers.get()

### PR DESCRIPTION
**Pull request by mdmintz**
### Description

Replace response.getheader() with response.headers.get() - (deprecated in urllib3 1.26.13)

### Motivation and Context

This will prevent lots of deprecation warnings when using selenium with the latest ``urllib3``

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: py/CHANGES [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Selenium (TBA version)
```
Comment (symonk) [inline py/CHANGES:1]: minor: I think we are gonna go with a `4.6.1` here, can you update that? could I also ask you to add a line about the `__del__` changes too, would be greatly appreciated

File: py/selenium/webdriver/remote/remote_connection.py [hunk 2]
```diff
@@ -319,23 +319,17 @@ def _request(self, method, url, body=None):
...
             if 300 <= statuscode < 304:
-                return self._request("GET", response.getheader("location"))
+                return self._request("GET", response.headers.get("location", None))
             if 399 < statuscode <= 500:
                 return {"status": statuscode, "value": data}
             content_type = []
-            if response.getheader("Content-Type"):
-                content_type = response.getheader("Content-Type").split(";")
+            if response.headers.get("Content-Type", None):
+                content_type = response.headers.get("Content-Type", None).split(";")
```
Comment (symonk) [inline py/selenium/webdriver/remote/remote_connection.py:332]: need to handle the `None` case here I think, will raise an `AttributeError` trying to split `NoneType` on some cases? edit: its already checked above ignore, no walrus allowed with py3.7+ support unfortunately which makes this kinda stuff much cleaner to code up.

**Review by symonk — APPROVED** [review]

**Comment by symonk** [comment]
Thank you @mdmintz

**Comment by Crocmagnon** [comment]
I hope we'll be able to have a release with this fix soon :)

**Pull request merged**